scripts/plots/nfcens/exevs_nfcens_wave_grid2obs_plots.sh

```diff
@@ -172,34 +194,6 @@ if [ $SENDDBN = YES ]; then
 	done
 fi
 
-
-#########################################
-# copy log files into logs and cat them
-########################################
-
-cd ${DATA}
-mkdir -p ${DATA}/logs
-log_dir=$DATA/logs
-
-extns='out log'
-for extn in ${extns} ; do
-	count=$(find ${DATA} -type f -name "*.${extn}"|wc -l)
-	if [ $count != 0 ] ; then
-		cp ${DATA}/*.${extn} ${log_dir}
-
-	fi
-done	
-
-
-log_file_count=$(find ${DATA} -type f -name "*.out" -o -name ".log" |wc -l)
-if [[ $log_file_count -ne 0 ]]; then
-	for log_file in $log_dir/*; do
-		echo "Start: $log_file"
-		cat $log_file
-		echo "End: $log_file"
-	done
-fi
-
```

Comment:
@SamiraArdani-NOAA @PerryShafran-NOAA Did we decide that the cat'ing sections need to stay in the plotting scripts because the plotting scripts do not use MET/METplus and therefore the changes to machine.conf do not impact them? If so, I think the cat'ing sections need to be added back to the NFCENS plotting scripts. 

CCing @ShelleyMelchior-NOAA

Reply:
They do need to stay in the plot script, because METplus is not used for the plotting portion.  I thought she had put them in, so either she actually didn't, or she put them in but then somehow later found their way out.

@SamiraArdani-NOAA Please be sure we keep the cat part of the *plot* stuff removed, but not for stats.

Reply:
@PerryShafran-NOAA : I just need to commit the changes. Also, what is your decision about the warning messages? I have added one line for each model in prep to echo the WARNING regardless of $SENDMAIL setting status.

Reply:
@SamiraArdani-NOAA If there is data missing, there must be a WARNING message in your output denoting the warning.  Since NCO does set SENDMAIL to NO, the warning message must be outside the SENDMAIL block.  The missing data must not cause any ERROR downstream in any way.  For example, if you have missing data in prep, it must not cause the stats job to crash.  All your codes need file checking, and a warning message if missing.  

@AliciaBentley-NOAA